style.css

```diff
@@ -0,0 +1,99 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@600&display=swap');
+@import url('https://fonts.cdnfonts.com/css/poppins');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background-repeat: no-repeat;
+  font-family: inter, sans-serif;
+}
+
+.topnav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  min-height: 48px;
+  background-color: #fff;
+  margin: 0 25px;
+}
+
+.logo {
+  font-family: poppins, sans-serif;
+  font-style: bold;
+  font-size: 18px;
+  line-height: 20px;
+  line-height: 74%;
+  font-weight: 700;
+  color: #6070ff;
+}
+
+.hambuger {
+  display: block;
+}
+
+.hambuger-bar {
+  width: 26px;
+  height: 3px;
+  background-color: #6070ff;
+  margin: 5px;
+}
+
+.container {
+  background-image: url("./images/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  min-height: 718px;
+}
+
+.first-section {
+  display: flex;
+  flex-direction: column;
+  margin: 0 24px;
+  padding-top: 155px;
```

Comment:
[OPTIONAL] I think it would be better to use responsive units for your ``margin``, ``padding``, ``fonts`` e.g **vh, svh, vw, %, etc** rather than using fixed **px** units, as with pixels it is easy to lose the design you worked for when the screen size changes 😉